src/capitalsFirst.js

```diff
@@ -17,8 +17,18 @@
  *
  * @returns {string} - string with uppercase words in front
  */
+
 function capitalsFirst(str) {
-  // write code here
+  const lowerCase = str.split(' ')
+    .filter(item =>
+      item.charAt(0) !== item.charAt(0).toUpperCase());
+
+  const upperCase = str.split(' ')
+    .filter(item =>
+      item.charAt(0) === item.charAt(0).toUpperCase());
+
+  return [...upperCase, ...lowerCase].filter(item =>
```

Comment:
this filtering is better done in the very beginning so that next actions are done with new filtered array